packages/react-ui-ag/src/Listings/SingleFamilyMobileMapListing.js

```diff
@@ -42,10 +42,14 @@ export default class SingleFamilyMobileMapListing extends PureComponent {
 
   @autobind
   handleButtonClick(onClick) {
-    const { prioritizeCardClick } = this.props
-
     return event => {
-      if (!prioritizeCardClick && onClick) onClick(this.props.listing)
+      const { prioritizeCardClick } = this.props
+
+      if (!prioritizeCardClick && onClick) {
+        onClick(this.props.listing)
+      } else if (prioritizeCardClick) {
```

Comment:
I think you need another check in this else if to check if the onClick prop exists